### PR TITLE
Remove CHEBI role classes from table

### LINKED ISSUE #87: Fix chemical stimulus trait pattern issues
List of tasks:
- [x] Remove the [CHEBI:50906 role](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_50906) classes from filler table [`response_to_chemical_stimulus_trait.tsv`](https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/patterns/data/default/response_to_chemical_stimulus_trait.tsv).
- [x] Replace some of the chemical entities in [`response_to_chemical_stimulus_trait.tsv`](https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/patterns/data/default/response_to_chemical_stimulus_trait.tsv) with chemical entities that are better matched to the GO response terms.
    - CHEBI:30879	alcohol --> CHEBI:16236	ethanol
- [x] Add new response to stimulus by role of chemical template and filler table. Use the existing OBA classes that are removed from the `response to chemical stimulus by chemical entity` template.
- [x] Add variables for `Contributors` to the yaml pattern template.
    - Use a data_list_var + annotation axiom that references it.
    - Use `dcterms:contributor`.
- [x] Check and make sure that all the filler tsv fields (i.e. contributors) work as intended.

See also related issue: <https://github.com/obophenotype/bio-attribute-ontology/issues/53#issuecomment-1111239926_>

### PR DESCRIPTION
**Pull request by rays22**
If applied, this commit will fix #87.

### CODE REVIEW

**Comment by matentzn** [comment]
Wait, we cant merge this - these classes will disappear! They should be stuck into the "chebi role" pattern before this can be merged!

**Comment by rays22** [comment]
> Wait, we cant merge this - these classes will disappear! They should be stuck into the "chebi role" pattern before this can be merged!

Yes, that was my plan too 😺 !

File: src/patterns/data/default/response_to_chemical_stimulus_trait.tsv
Comment (matentzn) [inline src/patterns/data/default/response_to_chemical_stimulus_trait.tsv:2]: What about the other orcids? Seems like you are now only allowing one?

File: src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml
Comment (matentzn) [inline src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml:6]: Mouthful... :D

File: src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml
Comment (matentzn) [inline src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml:52]: Does this reflect how GO does it?

I would prefer additional redundant () wherever possible, so 

```
('response_to_stimulus' and ('has_input' some %s))
```

File: src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml
Comment (matentzn) [inline src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml:28]: You didnt figure out to do data_list_vars eh? You should send a message to Huseyin and ask him how to do it

**Review by matentzn — CHANGES_REQUESTED** [review]
Looks great. Please run 

```
sh run.sh make ../patterns/definitions.owl
```

And commit changes to this branch!

File: src/patterns/data/default/response_to_chemical_stimulus_trait.tsv
Comment (rays22) [reply src/patterns/data/default/response_to_chemical_stimulus_trait.tsv:2]: Until the technical issue of adding orcids as data_list_vars is resolved, I can only add one orcid. I will make a ticket and raise this at the tech. team meeting on Monday.

Comment (matentzn) [reply src/patterns/data/default/response_to_chemical_stimulus_trait.tsv:2]: Excellent thank you!

Comment (rays22) [reply src/patterns/data/default/response_to_chemical_stimulus_trait.tsv]: Contributor ORCIDs are fixed now.

File: src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml
Comment (rays22) [reply src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml]: Done.

File: src/patterns/definitions.owl
Comment (matentzn) [inline src/patterns/definitions.owl:47947]: orcid should not be a string, it should be an IRI!

File: src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml
Comment (matentzn) [inline src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml:59]: This does not seem correct.. Shouldnt it be 'has_input' some ('has_role' some %)? Else how is this pattern different than the other one?

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml
Comment (matentzn) [inline src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml:60]: Just to be fully aligned with upheno:

https://raw.githubusercontent.com/obophenotype/upheno/master/src/patterns/dosdp-dev/abnormalLevelOfChemicalEntityWithRoleInLocation.yaml

```
('has_input' some ('chemical entity' and ('has_role' some %s)))))
```
?

File: src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml
Comment (matentzn) [inline src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml:36]: remaining this to figure out.. Should transform to `<https://orcid.org/0000-0002-7356-1779>` in RDF, not `"https://orcid.org/0000-0002-7356-1779"^^xsd:string`

**Review by matentzn — CHANGES_REQUESTED** [review]

**Pull request merged**
